.github/workflows/validate.yml

```diff
@@ -35,7 +36,15 @@ jobs:
         with:
           useLockFile: false
 
-      - name: ▶️ Run validate script
+      - name: Install ESLint v${{ matrix.eslint }}
+        run: npm install --no-save --force eslint@${{ matrix.eslint }} --legacy-peer-dependencies
```

Comment:
Have you checked to see if we can drop the `--legacy-peer-dependencies`? If we can get away with not having it, I think that'd be good to help us catch any future problems with peer dependencies (but I'm not fussed if we do need it)

Reply:
Even without the `--legacy-peer-dependencies`, CI is still green 🥳🎉